kv/kv_interface.go

```diff
@@ -298,6 +298,7 @@ type StatelessRwTx interface {
 //   - ReadOnly transactions do not lock goroutine to thread, RwTx does
 type Tx interface {
 	StatelessReadTx
+	BucketMigratorRO
```

Comment:
can add method ListBuckets() rigth to Tx interface, without creating BucketMigratorRO

Reply:
I did that initially, but turned out `BucketMigrator` also needs it